rollup.config.js

```diff
@@ -111,6 +112,11 @@ const baseConfig = {
     resolve({
       extensions,
     }),
+    postcss({
+      extract: 'components.css',
+      autoModules: false,
+      modules: {generateScopedName: 'prc_[local]-[hash:base64:5]'},
+    }),
```

Comment:
Is it possible with the plugin to emit outside of the target directory? e.g. instead of emitting both to `lib-esm/components.css` and `lib/components.css` could we emit to `css/components.css`?

Reply:
I think it should be possible. I'll try it :)